File: RSPAClient/Core/AtPar.ViewModel/VM_VIEW_CART.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_VIEW_CART
    {
        public int ID { get; set; }
        public string ITEM_ID { get; set; }
        public string ITEM_DESCR { get; set; }
        public string SERIAL_NO { get; set; }
        public Nullable<int> REQUEST_QTY { get; set; }
        public string LOCATION_ID { get; set; }
        public Nullable<System.DateTime> ESTIMATED_RETURN_DATE { get; set; }
        public Nullable<int> ITEM_QTY { get; set; }
        public string ITEM_TYPE_INDICATOR { get; set; }
        public string IMAGE { get; set; }
        public Nullable<System.DateTime> NEEDED_BY_DATE { get; set; }
        public string PATIENT_ID { get; set; }
        public string PATIENT_LAST_NAME { get; set; }
        public string PROCEDURE_CODE { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_VIEW_CART.ts
export class VM_VIEW_CART {
    public ID: number;
    public ITEM_ID: string;
    public CHK_VALUE: number;
    public ITEM_TYPE_INDICATOR: string;
    public ITEM_DESCR: string;
    public ITEM_QTY?: number;
    public IMAGE: string;
    public LOCATION_ID: string;
    public SERIAL_NO: number;
    public PATIENT_ID: string;
    public PATIENT_LAST_NAME: string;
    public REQUEST_QTY: number;
    public DUE_DATE: any;
    public RETURN_DATE: any;
    public NEEDED_BY_DATE: any;
    public ESTIMATED_RETURN_DATE: any;
    public PROCEDURE_CODE: string;
    public DUE_TIME: any;
    public RETURN_TIME: any;
    public BMPIMAGE: string;

    public blnShowItemImage: boolean = true;
    public blnShowQtyAvailable: boolean = true;
    public blnShowSelectQty: boolean = true;
    public blnShowDueDateTime: boolean = true;
    public blnShowAvailableTo: boolean = true;
    public blnShowReturnDateTime: boolean = true; 
}